app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
Just in accordance with the previous comment kindly make the necessary adjustment here😉.

```suggestion
  belongs_to :author, class_name: 'User'
```